pkg/controllers/management/auth/crtb_handler.go

```diff
@@ -68,10 +68,7 @@ const (
 
 var clusterManagementPlaneResources = map[string]string{
 	"clusterscans":                "management.cattle.io",
-	"catalogtemplates":            "management.cattle.io",
-	"catalogtemplateversions":     "management.cattle.io",
-	"clustercatalogs":             "management.cattle.io",
-	"clustermonitorgraphs":        "management.cattle.io",
+	"clusterloggings":             "management.cattle.io",
```

Comment:
Guessing this is a rebase/merge hiccup. I removed this here https://github.com/rancher/rancher/pull/47997 so could you remove `clusterloggings`?